client/src/components/Onboarding/OnboardingAdmin.js

```diff
@@ -1,7 +1,357 @@
+              }
+              disabled={adminAction.loading}
+              disableSkip={step.state === 'completed'}
+              sentAction={
+                adminAction.username === user.username &&
+                adminAction.step === step.step
+                  ? adminAction.action
+                  : null
+              }
+            />
+          </div>
+        ))}
+      </td>
+      <td>
+        {user.steps.map(step => (
+          <div
+            key={uuidv4()}
+            styleName={step.state === 'staffwait' ? 'staffwait' : ''}
+          >
+            <Button
+              color="link"
+              styleName="action-link"
+              onClick={() => viewLogCallback(user, step)}
+            >
+              View Log
+            </Button>
+          </div>
+        ))}
+      </td>
+    </tr>
+  );
+};
+
+OnboardingAdminListUser.propTypes = {
+  user: onboardingUserPropType.isRequired,
+  viewLogCallback: PropTypes.func.isRequired
+};
+
+const OnboardingAdminList = ({ users, viewLogCallback }) => {
+  return (
+    <table styleName="users">
+      <thead>
+        <tr>
+          <th>User</th>
+          <th>Step</th>
+          <th>Status</th>
+          <th>Administrative Actions</th>
+          <th>&nbsp;</th>
+          <th>Log</th>
+        </tr>
+      </thead>
+      <tbody>
+        {users.map(user => (
+          <OnboardingAdminListUser
+            user={user}
+            key={user.username}
+            viewLogCallback={viewLogCallback}
+          />
+        ))}
+      </tbody>
+    </table>
+  );
+};
+
+OnboardingAdminList.propTypes = {
+  users: PropTypes.arrayOf(onboardingUserPropType).isRequired,
+  viewLogCallback: PropTypes.func.isRequired
+};
+
+const OnboardingAdmin = () => {
+  const dispatch = useDispatch();
+  const [eventLogModalParams, setEventLogModalParams] = useState(null);
+
+  const { users, offset, limit, total, query, loading, error } = useSelector(
+    state => state.onboarding.admin
+  );
+
+  const paginationCallback = useCallback(
+    page => {
+      dispatch({
+        type: 'FETCH_ONBOARDING_ADMIN_LIST',
+        payload: {
+          offset: (page - 1) * limit,
+          limit,
+          query
+        }
+      });
+    },
+    [offset, limit, query]
+  );
+
+  const viewLogCallback = useCallback(
+    (user, step) => {
+      setEventLogModalParams({ user, step });
+    },
+    [setEventLogModalParams]
+  );
+
+  const toggleViewLogModal = useCallback(() => {
+    setEventLogModalParams();
+  }, [setEventLogModalParams]);
+
+  useEffect(() => {
+    dispatch({
+      type: 'FETCH_ONBOARDING_ADMIN_LIST',
+      payload: { offset, limit, query: null }
+    });
+  }, [dispatch]);
+
+  const current = Math.floor(offset / limit) + 1;
+  const pages = Math.ceil(total / limit);
+  if (loading) {
+    return <LoadingSpinner />;
+  }
+  if (error) {
+    return (
+      <Message type="warn">Unable to access Onboarding administration</Message>
```

Comment:
Any way to center this vertically and horizontally in the view?
<img width="1001" alt="Screen Shot 2021-03-18 at 4 44 01 PM" src="https://user-images.githubusercontent.com/20326896/111706538-73e25880-8810-11eb-957e-28f6074a7b2d.png">

Reply:
Yes.

Reply:
```

Reply:
I've fiddled around with this. There are obstacles.

- Horizontally centering the message will mean horizontally centering any `.workbench-content` content (`align-items: center;`).
- A `.workbench-content` `:only-child` content is set to stretch to fix a layout bug, which means the message is super tall, so it can't be vertically-centered via new style for `.workbench-content` (`justify-content: center;`).
    - This fix _may_ be outdated, but I'd want to confirm that before I take it away.

I can hack some Onboarding-specific global CSS to do the job (`.workbench-content > :only-child { height: auto; }`), but I would be solving the wrong problem.

What is likely more desirable is one of these options:
- showing this message _within_ the section (rather than in place of the section)
- having a standard solution (from Design) for disallowing access to sections
- (some other idea)

Reply:
[I ask Design now](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1616183944001000), cuz maybe they want just a vertically/horizontally centered message. If so, then. it should be a `<SectionMessage>` (not an `<inlineMessage>`; **warning**: this change will make the "stretch"—from [previous comment]—uncomfortably obvious).

Reply:
Based on [Slack discussion](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1616183944001000), we will revisit this in a new ticket.

Reply:
New Ticket
[FP-1026: Use Section Message for Onboarding Access Warning](https://jira.tacc.utexas.edu/browse/FP-1026)